cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -252,20 +244,24 @@
 default['travis_build_environment']['neo4j_checksum'] = '24fd6a704e0d80c4b4f9a3d17ce0db23f258a8cdcfa1eb28d7803b7d1811ee96'
```

Comment:
I noticed that neo4j has a new update out too - 3.2.2 - though understand if you'd rather not touch it in this PR :-)